src/frontend/screens/CategoryChooser.js

```diff
@@ -71,20 +71,52 @@ const Item = React.memo(
 const CategoryChooser = ({ navigation }: { navigation: NavigationProp }) => {
   const [{ presets }] = useContext(ConfigContext);
   const [{ value: draftValue }, { updateDraft }] = useDraftObservation();
-
   const presetsList = Array.from(presets.values())
     // Sort presets by sort property and then by name, then filter only point presets
     .sort(presetCompare)
     // Only show presets where the geometry property includes "point"
     .filter(p => p.geometry.includes("point"));
-
   const handleSelectPreset = (preset: Preset) => {
-    updateDraft({
-      tags: {
-        ...(draftValue || {}).tags,
-        categoryId: preset.id,
-      },
-    });
+    const newTags = preset.tags;
```

Comment:
As a matter of code style, I think it's better to just use `preset.tags` to refer to this throughout, since assigning it to a new name (`newTags`) just means anyone reading the code has to go back through the code to check what it refers to (here, `newTags` and `preset.tags` are referring to the same value).